.github/workflows/macos.yml

```diff
@@ -129,6 +129,7 @@ jobs:
 
       - name: 📦 Package
         run: |
+          echo "Killing XProtect..."; sudo pkill -9 XProtect >/dev/null || true; # see https://github.com/actions/runner-images/issues/7522
```

Comment:
@m-kuhn , do we need this for iOS?

Reply:
No, it's the DMG creation -- specifically on macos-13 runners.